packages/buckets/src/buckets.ts

```diff
@@ -348,6 +373,31 @@ export class Buckets extends BucketsGrpcClient {
    *    return await buckets.pushPath(bucketKey!, 'index.html', file)
    * }
    * ```
+   * 
+   * @example
+   * Push an folder in node.js
```

Comment:
duplicate example. with example checking though, shouldn't be issue keeping updated